### DISCUSSION TITLE
wire:loading modifiers

### DISCUSSION THREAD

**Discussion by oxytoxin** [livewire/livewire#2650] [4. General]
Hi! Is it not possible to chain modifiers such as doing wire:loading.grid.delay?
Thanks a bunch!

**Comment by nuernbergerA** [comment]
Consider toggling the class instead
https://laravel-livewire.com/docs/2.x/loading-states#toggling-classes

**Comment by oxytoxin** [comment]
Thanks, the result was still the same but I managed to solve it by wrapping the children elements into a grid container itself while the the loading component remained inline-block.
